### PR TITLE
Create To-Do-List list structure

### PR DESCRIPTION
**Pull request by tasnimchy96**
In this branch we :

Create a new project directory and navigate into it.
Initialize a new npm project by running npm init -y in the terminal.
Install webpack and webpack-cli by running npm install webpack webpack-cli --save-dev.
Create an index.html file in the project root directory and write your HTML markup there. Include a placeholder element (e.g.,
or
) for the To-Do list.
Install the HTML Webpack Plugin by running npm install html-webpack-plugin --save-dev.
Create an index.js file in the project root directory. In this file, define an array of simple to-do tasks, where each task is represented by an object with three keys: description (string), completed (boolean), and index (number).
Write a function in index.js to iterate over the tasks array and populate an HTML list item element for each task. Use the DOM manipulation methods to create and append the list item elements to the placeholder element in the HTML.
Install the Webpack Style/CSS Loader and CSS Loader by running npm install style-loader css-loader --save-dev.
Create a style.css file and define the necessary CSS rules to style your To-Do list. Make sure the CSS file is loaded by Webpack using the style-loader and css-loader.
Create a webpack.config.js file in the project root directory. Configure the entry point as index.js, and specify the output path and filename.
Configure the HTML Webpack Plugin in the webpack.config.js file to use index.html as the template and inject the bundled JavaScript file.
Configure the Style/CSS Loader in the webpack.config.js file to handle CSS files.
Build the project by running npx webpack in the terminal.
Serve the project using a development server (e.g., npx webpack serve) or by opening the index.html file in a browser.

### CODE REVIEW

**Review by aamirkhan2478 — COMMENTED** [review]
Hi @tasnimchy96, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request :red_circle: :red_circle: :red_circle: 

You have submitted a project, that hasn't config the linter correctly kindly config your linters correctly and then re-submit :slightly_smiling_face: 

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: src/style.css [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+  height: 20px;
+  color: black;
+  background-color: #fff;
+}
+
+.clear-completed {
+  width: 100%;
+  border: none;
+  background-color: transparent;
+  padding: 5px;
```
Comment (usorfaitheloho) [inline src/style.css:81]: Kindly note that your list should represent a minimalist design replicate that was given, all features don't need to be implemented now but it should at least resemble the framework. Please work on your styling to an extent as it is really different from what is expected 😄 

Your design :
image

Design given :
image

**Review by usorfaitheloho — CHANGES_REQUESTED** [review]
> Hi @tasnimchy96  ,
> 
> Good job so far! There are some issues that you still need to work on to go to the next project but you are almost there!

  [ ](https://camo.githubusercontent.com/679757134a8fee76cfe1941f9f3ecb0e0cf56dd02da4498268ce31f0dacdfca3/68747470733a2f2f6d65646961302e67697068792e636f6d2f6d656469612f556461535559436c723932636a57706556782f67697068792e676966)
> 
> ## Highlights
> * Webpack is configured ✔️
> * Descriptive pull request ✔️
> * Used GitHub flow ✔️
> 
> ## Required Changes ♻️
> _Check the comments under the review._
> 
> ## Optional suggestions
> _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
> 
> Cheers and Happy coding!👏👏👏
> 
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.
> 
> **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
> 
> _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/style.css (hunk 1 above)
Comment (tasnimchy96) [reply src/style.css:81]: done

Comment (Lucas-Masaba) [reply src/style.css:81]: - [ ] Please make the changes ass requested by the previous reviewer. Your project's UI should resemble that provided by the [wireframe](https://www.youtube.com/watch?v=AcUd-_Yjjqg). Just so you know, the video has more features than the current project. You don't need to style all the features not implemented yet in the video. Currently, you UI still looks like this:
image

**Review by Lucas-Masaba — CHANGES_REQUESTED** [review]
# Hello @tasnimchy96  👋 , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.
almost-there

 ## Highlights 🍾 
- Linter checks passing :heavy_check_mark: 

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions 💭 

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏 💻 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @Lucas-Masaba in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by usorfaitheloho — APPROVED** [review]
Hi @tasnimchy96  , 
# STATUS: APPROVED! 🥇 💯 

done

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**